src/main/java/eiteam/esteemedinnovation/materials/raw/config/OreConfigurationParser.java

```diff
@@ -246,7 +246,7 @@ private void writeDefault() throws IOException {
                 path = pieces[1];
             } else {
                 if (pieces.length > 3) {
-                    FMLLog.warning("[EI] More than 3 values separated by : in ReplaceableBlocks '%s' for Ore generation config. Expected maximum: 'domain:path:meta'. This may indicate a bug in your config!", str);
+                    EsteemedInnovation.logger.warn("[EI] More than 3 values separated by : in ReplaceableBlocks '%s' for Ore generation config. Expected maximum: 'domain:path:meta'. This may indicate a bug in your config!", str);
```

Comment:
`[EI]` should be removed from this, because it says `[esteemedinnovation]` in each log entry